dateparser/timezones.py

```diff
@@ -6,7 +6,7 @@
 timezone_info_list = [
     {
         'regex_patterns':
-            [r'(\W|\d|_)\(?%s\)?$'],
+            [r'(\W|\d|_)\(?%s(\)|$|[^\w+])'],
```

Comment:
I think we should not rush this and think for a while. May be move timezone extraction to a level higher.